pkg/util/ecs/metadata/clients.go

```diff
@@ -20,6 +22,7 @@ import (
 )
 
 var globalUtil util
+var cloudProviderName = "AWS"
```

Comment:
can we define this const once? it's already here https://github.com/DataDog/datadog-agent/blob/master/pkg/util/ecs/detection.go#L26

Reply:
Makes sense, missed that 👍

Reply:
For defining it once, do I have to redefine the const in the `metadata` package? I know it is defined in the parent package `ecs`, but if I import that, I get an import cycle.

Reply:
you can still create a new pkg `common` under `util/ecs` and define the const in it and import it from there in both metadata and detection.go

this is still not ideal IMHO, we should have had dedicated `ConfigProviderNameType` (string) defined in the config package and covers all cloud provider types, so we can avoid import cycles and more importantly avoid invalid inputs and typos. I do understand that this could be out of scope for this PR

Reply:
Should I create a card about the latter in the Backlog?

Reply:
Created one here: https://trello.com/c/ZW8Nenmu/1903-create-a-dedicated-cloud-provider-name-type-in-the-config-package